DuckDuckGo/TabViewController.swift

```diff
@@ -867,6 +872,16 @@ extension TabViewController: WKNavigationDelegate {
     
     private func onWebpageDidFinishLoading() {
         os_log("webpageLoading finished", log: generalLog, type: .debug)
+        
+        let trackersWorkItem = DispatchWorkItem {
```

Comment:
This is biggish chunk of related code, how about extracting this to a private function?